Translations/translation_EN.json

```diff
@@ -1,207 +1,339 @@
+		"SettingOffChar": "O",
+		"SettingSlowChar": "S",
+		"SettingMediumChar": "M",
+		"SettingFastChar": "F",
+		"SettingStartNoneChar": "O",
+		"SettingStartSolderingChar": "S",
+		"SettingStartSleepChar": "Z",
+		"SettingStartSleepOffChar": "R",
+		"SettingSensitivityOff": "O",
+		"SettingSensitivityLow": "L",
+		"SettingSensitivityMedium": "M",
+		"SettingSensitivityHigh": "H",
+		"SettingLockDisableChar": "D",
+		"SettingLockBoostChar": "B",
+		"SettingLockFullChar": "F"
+	},
+	"menuGroups": {
+		"PowerMenu": {
+			"text2": [
+				"Power",
+				"settings"
+			],
+			"desc": "Power settings"
+		},
+		"SolderingMenu": {
+			"text2": [
+				"Soldering",
+				"settings"
+			],
+			"desc": "Soldering settings"
+		},
+		"PowerSavingMenu": {
+			"text2": [
+				"Sleep",
+				"mode"
+			],
+			"desc": "Power saving settings"
+		},
+		"UIMenu": {
+			"text2": [
+				"User",
+				"interface"
+			],
+			"desc": "User interface settings"
+		},
+		"AdvancedMenu": {
+			"text2": [
+				"Advanced",
+				"settings"
+			],
+			"desc": "Advanced settings"
+		}
+	},
+	"menuOptions": {
+		"DCInCutoff": {
+			"text2": [
+				"Power",
+				"source"
+			],
+			"desc": "Power source. Sets cutoff voltage. (DC 10V) (S 3.3V per cell, disable power limit)"
+		},
+		"SleepTemperature": {
+			"text2": [
+				"Sleep",
+				"temp"
+			],
+			"desc": "Tip temperature while in \"sleep mode\""
+		},
+		"SleepTimeout": {
+			"text2": [
+				"Sleep",
+				"timeout"
+			],
+			"desc": "Interval before \"sleep mode\" kicks in (s=seconds | m=minutes)"
+		},
+		"ShutdownTimeout": {
+			"text2": [
+				"Shutdown",
+				"timeout"
+			],
+			"desc": "Interval before the iron shuts down (m=minutes)"
+		},
+		"MotionSensitivity": {
+			"text2": [
+				"Motion",
+				"sensitivity"
+			],
+			"desc": "0=off | 1=least sensitive | ... | 9=most sensitive"
+		},
+		"TemperatureUnit": {
+			"text2": [
+				"Temperature",
+				"unit"
+			],
+			"desc": "C=Celsius | F=Fahrenheit"
+		},
+		"AdvancedIdle": {
+			"text2": [
+				"Detailed",
+				"idle screen"
+			],
+			"desc": "Display detailed information in a smaller font on the idle screen"
+		},
+		"DisplayRotation": {
+			"text2": [
+				"Display",
+				"orientation"
+			],
+			"desc": "R=right-handed | L=left-handed | A=automatic"
+		},
+		"BoostTemperature": {
+			"text2": [
+				"Boost",
+				"temp"
+			],
+			"desc": "Tip temperature used in \"boost mode\""
+		},
+		"AutoStart": {
+			"text2": [
+				"Start-up",
+				"behavior"
+			],
+			"desc": "O=off | S=heat up to soldering temp | Z=standby at sleep temp until moved | R=standby without heating until moved"
+		},
+		"CooldownBlink": {
+			"text2": [
+				"Cooldown",
+				"flashing"
+			],
+			"desc": "Flash the temperature reading after heating was halted while the tip is still hot"
+		},
+		"TemperatureCalibration": {
+			"text2": [
+				"Calibrate",
+				"Tip Offset?"
+			],
+			"desc": "Calibrate tip temperature offset; not required unless device > 5°C off target"
```

Comment:
You cant use `°` which is why the build is failing.
`°` Does not exist in the python font map

Reply:
thanks